notebooks/en/rag_with_sql_reranker.ipynb

```diff
@@ -0,0 +1,849 @@
+{
```

Comment:
<p>I think the code comment below is enough without needing to create another warning which can be distracting :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done.